### ISSUE TITLE
Aufgabenaufteilung und Umfang

### ISSUE THREAD

**Issue by a-schulz** [a-schulz/aai-selfdriving-cars#1]
- [x] requirements bereinigen (alte loeschen und neu pip freeze)
- [x] cli -> python programm
- [x] pretrained model um lanes erweitern (label-studio ka. iwie so heisst eine software dafuer) (jakob)
- [ ] traffic signs verschiedene typen hinzufuegen (alex)
- [x] #2
-> evtl muss das model nochmal komplett erneuert werden

=> proof of concept

- [ ] dokumentation -> readme.md
- [ ] statistiken (accuracy)
- vergleich vorher YOLO vorher mit finetune

yolo siehe= https://github.com/ultralytics/ultralytics

deadline =

**Comment by a-schulz** [comment]
Traffic light detection
https://www.kaggle.com/datasets/wjybuqi/traffic-light-detection-dataset

**Comment by jay-hae** [comment]
Wir müssen aus Zeitgründen wahrscheinlich das fintunen mit lanes machen. Dafür benötigt man ein gutes dataset. Des weiteren ist es schwer lanes nur mit bounding boxes zu kennzeichnen. Hier könnte die KI dann auf ungeahnte patterns trainiert werden. Da die Lanes an sich sehr klein in der bounding box wären. Um ein eigenes Dataset mit gut annotierten polygonen zu erstellen braucht man einfach zu lange. Ich würde daher den Punkt als erledigt an sehen und lieber nur die Traffic  light und sign klassifikation vornhemen.

**Comment by a-schulz** [comment]
Stand 02.08.2023

1. Phase = https://universe.roboflow.com/tu-wien-pfowz/traffic-sign-detection-yolov8/dataset/10
-> Erkennen von TrafficSigns
1.1 Falls gefunden -> erkennung der Klassen mittels ResNet (self trained)
2. Erkennung der restlichen Elemente mittels yolov8n
2.2 Erkennung der traffic lights nach den bestimmten phasen
3. Evlt nachträglich dann noch herausgeben von Aktions (je nach dem, was der Prof sagt)
